clouddriver-eureka/src/main/groovy/com/netflix/spinnaker/clouddriver/eureka/deploy/ops/AbstractEurekaSupport.groovy

```diff
@@ -264,36 +264,50 @@ abstract class AbstractEurekaSupport {
     return serverGroup
   }
 
+  /**
+   * Returns a list of instanceIds to disable. Only really used for RollingRedBlack strategy.
+   * The list represents the given percentage of "enabled" instances.
+   * Enabled instance is one that that has at least 1 health provider indicating it's UP
```

Comment:
seems technically correct, but maybe this no longer belongs in `AbstractEurekaSupport`?

Reply:
I didn't want to refactor/change too much I will move it around when i do more RRB stuff, how do you feel about that?